### PR TITLE
user profile

### PR DESCRIPTION
**Pull request by Helia-nik**
# Description

This PR add profile page.

## Related issues
Fixes [#145](https://quire.io/w/Caspian_Tamin/145/user_profile?view=board)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?
 Edit my profile and check it from profile page and database.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/components/layouts/admin-layout.js [hunk 1]
```diff
@@ -164,15 +164,29 @@ const AdminLayout = ({ children }) => {
...
+						<div className="flex justify-center items-center">
 							{entityName && (
 								<Link className="py-2 px-4 mr-4 text-center btn" to={`/${workspaceId}/${entityName}/create`}>
 									Add new
 								</Link>
 							)}
+							<Link className="px-2" to={'/'}>
+								Home
+							</Link>
+							<Link className="px-2" to={`/${workspaceId}/profile`}>
```
Comment (Yasaman-k) [inline src/components/layouts/admin-layout.js:176]: sharedworkspaceId also must add yes?

**Review by Yasaman-k — CHANGES_REQUESTED** [review]

File: src/components/layouts/admin-layout.js (hunk 1 above)
Comment (Helia-nik) [reply src/components/layouts/admin-layout.js:176]: nope, i check it there is no problem and its work.

Comment (Yasaman-k) [reply src/components/layouts/admin-layout.js:176]: if workspace has sharedworkspace ,is it also working?

Comment (Yasaman-k) [reply src/components/layouts/admin-layout.js:176]: Great !

**Review by Yasaman-k — APPROVED** [review]

**Pull request merged**